lib/sprockets/sass_importer.rb

```diff
@@ -1,2 +1,9 @@
-# Deprecated: Require sprockets/sass_processor instead
-require 'sprockets/sass_processor'
+
+require 'sprockets/autoload'
+
+module Sprockets
+  class SassImporter < Autoload::Sass::Importers::Filesystem
+    # NOTE: Hack to support sourcemap generation in Sass-3.3
+    def public_url(a, b);'';end
```

Comment:
I didn't find a usecase where it would be necessary, one thing it might be used for is the sourcemap comment in order to link to the original asset, but sprockets overrides this one anyway.